api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/allowExportSelectedData.md

```diff
@@ -5,13 +5,15 @@ default: false
 ---
 ---
 ##### shortDescription
-Allows an end user to export selected rows only.
+Allows users to export selected rows only.
 
 ---
 When this option is set to **true**, a click on <img src="/Content/images/doc/20_1/DataGrid/icons/toolbar_export.png" alt="DevExtreme DataGrid HTML5 Toolbar Exporting" style="vertical-align:middle"/> invokes a menu that contains the "Export selected rows" command.
 
 ![DevExtreme DataGrid Exporting Selected Rows](/images/DataGrid/Export_SelectedRows.png)
 
+[note]This option cannot be enabled when [custom server-side summaries](/Documentation/Guide/Widgets/DataGrid/Summaries/Custom_Aggregate_Function/#Server-Side_Data_Aggregation) are used.
+
 #####See Also#####
 - **export**.**texts**.[exportSelectedRows](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/export/texts/exportSelectedRows.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/texts/#exportSelectedRows') - customizes the text of the "Export selected rows" command.
-- **selection**.[mode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/selection/mode.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/selection/#mode') - enables selection in the widget.
\ No newline at end of file
+- **selection**.[mode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/selection/mode.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/selection/#mode') - enables selection in the widget.
```

Comment:
Can we make this description more specific?
For example, does it allow you to select rows to be exported?

Reply:
Not in this case, because this section contains links to related articles. This **selection**.mode is an article on selecting rows in DataGrid in general, not only when exporting.